src/components/Dialog.tsx

```diff
@@ -36,16 +46,15 @@ export default class Dialog extends Component {
         isOpen={isOpen}
         className={"madeUpClassToCancelDefaultStyles"}
         onRequestClose={closeFn}
-        close={closeFn}
```

Comment:
Removed this prop. I think it got superseded by `onRequestClose` at some point and was accidentally left in there.